docs/source/tutorials/cascaded_architecture.rst

```diff
@@ -0,0 +1,144 @@
+Cascaded architecture
+=====================
+
+In this tutorial we will learn the following features:
+
+- Design a training scheme composed of two cascaded networks.
+- Visualize the training with tensorboard
+- Find the optimal threshold to binarize images based on the validation sub-dataset.
+
+In our example, the model will first locate the spinal cord (step 1). This localisation will then be used to crop the images around this region of interest, before segmenting the cerebrospinal fluid (CSF, step 2).
+
+.. _Prerequisite:
+
+Prerequisite
+------------
+
+In this tutorial, the spinal cord segmentation model generated from :doc:`../tutorials/one_class_segmentation_2d_unet`
+will be needed since it will be used for the step 1. Please make sure you did this tutorial prior to this present example and that a folder named ``seg_sc_t1_t2_t2s_mt`` is available.
```

Comment:
should we use "seg_sc_t1-t2-t2s-mt" as per [our conventions](https://github.com/ivadomed/ivadomed/wiki/repositories#models)?

Reply:
Yes, absolutely. I fixed it here fd210e2